### DISCUSSION TITLE
PS methodology requires no multilook approach

### DISCUSSION THREAD

**Discussion by vittorio3d** [alexisInSAR/EZ-InSAR#76] [Ideas and further developments]
Hi everyone, I noticed that by selecting SLC stack processing it is not possible to set azimuth and range and by default the values - r 9 and - z 3 are assigned therefore with a multilook approach but from what I know the ps are extracted correctly with a full resolution approach (-r 1 - z 1). I don't think setting input_file solves the problem

**Comment by alexisInSAR** [comment]
Hi @vittorio3d, 

EZ-InSAR does not work in this way. An ISCE processing generates two levels of files: for example, for the elevation file in radar coordinates :
- hgt.hdr; 
- hgt.rdr; 
- hgt.rdr.aux.xml; 
- hgt.rdr.full; 
- hgt.rdr.full.aux.xml; 
- hgt.rdr.full.vrt; 
- hgt.rdr.full.xml; 
- hgt.rdr.hdr; 
- hgt.rdr.vrt. 

The *full* data are in full resolution. 

Then, when you run a StaMPS processing, the script named @[run_SLCcropStack_mod.csh] (https://github.com/alexisInSAR/EZ-InSAR/blob/Version_2_2_1_Beta/EZINSAR_BIN/Suppfunctions/run_SLCcropStack_mod.csh) will only use files whose name contains the string *full*. 

Same behaviour for the ISCE script: crop_rdr.py: the default parameters are full. 

Kind regards, 

Alexis
